src/pages/openSourceApply.js

```diff
@@ -88,7 +88,7 @@ class OpenSourceApplyPage extends Component {
     const { repositories } = this.state;
 
     if (!LoggedInUser) {
-      return <SignInOrJoinFree />;
+      return <SignInOrJoinFree redirect={Router.asPath} initialForm="create-account" />;
```

Comment:
We had problem in the past with `Router.asPath` (see https://github.com/opencollective/opencollective-frontend/pull/2039) it breaks because Router instance is not available in SSR. However `SignInOrJoinFree` is clever enough to use the current page for redirect if none is provided.

```suggestion
      return <SignInOrJoinFree initialForm="create-account" />;
```

Reply:
I actually had to revert this thing because it was breaking the switch between the forms. I'll create another issue for it.

Reply:
Argh, seems like I need it for the `contributeAs` flow after all.
I'll remove the switch using the url. It's not really needed, especially since the default form for `/signin` will be create an account.